drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/main/java/org/drools/workbench/screens/dtablexls/backend/server/conversion/GuidedDecisionTablePopulater.java

```diff
@@ -116,70 +118,131 @@ private void processIndirectSourceBuilders(final int maxRowCount) {
     }
 
     private void addIndirectSourceBuildersColumns() {
-        final List<BRLVariableColumn> variableColumns = new ArrayList<BRLVariableColumn>();
+        //Extract BRLVariableColumn and SourceBuilders for respective potions of the rule
```

Comment:
potions?

Reply:
Typo.. should have been "portions" but I've changed to "sections"!